src/components/BuySell/components/BuySellSelector.tsx

```diff
@@ -3,13 +3,19 @@ import styled from 'styled-components'
 
 import useBuySell from 'hooks/useBuySell'
 
-const BuySellSelector: React.FC = () => {
+interface BuySellSelectorProps {
+  tokenId: any
+}
+
+const BuySellSelector: React.FC<BuySellSelectorProps> = ({ tokenId }) => {
```

Comment:
instead of passing `tokenId` through the components above, you can access the currently viewing token via the `useBuySell` hook below.
e.g. `const { buySellToken } = useBuySell()`